Source/WebKit/Shared/Cocoa/SandboxExtensionCocoa.mm

```diff
@@ -321,6 +321,21 @@ String resolvePathForSandboxExtension(StringView path)
     return WTFMove(handle);
 }
 
+#if HAVE(SANDBOX_STATE_FLAGS)
+static void enableMachBootstrap(std::optional<audit_token_t> auditToken)
+{
+    if (!WTF::processHasEntitlement("com.apple.private.security.message-filter"_s))
+        return;
```

Comment:
Under what conditions is the entitlement missing?

Why is the entitlement required? That would seem to limit the scope of this important security feature.

Reply:
Yes, that is a good point. We are no longer relying on sandbox state variables, but instead using sandbox extensions, which will enable this feature for all clients.

Thanks for reviewing!